src/github.com/getlantern/tlsdialer/tlsdialer.go

```diff
@@ -51,21 +44,6 @@ type ConnWithTimings struct {
 	VerifiedChains [][]*x509.Certificate
 }
 
-// OverrideResolve allows overriding the DNS resolution function
-func OverrideResolve(override func(addr string) (*net.TCPAddr, error)) {
```

Comment:
This functionality has been superseded by netx.  Note how the old tlsdialer code didn't have a mutex on the global resolve and dialOverride fields.  This could theoretically have led to a race condition that showed the error @atavism  was seeing.  netx uses atomic.Value for these sorts of things, so that shouldn't be a problem.